README.md

```diff
@@ -117,7 +117,7 @@ You can overwrite Rollbar configuration in environment's config. Here is the def
       javascript: {
         source_map_enabled: true,
         guess_uncaught_frames: true
-        // code_version: "some version string, such as a version number or git sha",
+        code_version: "YOUR_APP_VERSION" // returns app version in format: 2.4.0+06df23a
```

Comment:
From my perspective, hash commit is not so important here, especially, that in the production environment I use tagged version for deploy purpose (if you deploy head of the master branch that wasn't released yet, then probably something is not OK in the other words)

Reply:
The point here is that many commits can have the same app version in package.json. I think you just mislead it with Github Releases. 
So I can deploy many times (eg. staging) with the same version (as it is in tests right now) but  I'll still be able to distinguish the changes based on commit hash